app/serializers/dependents_application_serializer.rb

```diff
@@ -1,7 +1,10 @@
 # frozen_string_literal: true
 
-class DependentsApplicationSerializer < ActiveModel::Serializer
-  attribute(:guid)
-  attribute(:state)
-  attribute(:parsed_response)
+# NOTE: I don't think is being used anywhere
```

Comment:
Do you want a review from @department-of-veterans-affairs/vfs-authenticated-experience-backend?

Reply:
I think they should. If it's being used, then I can delete it (and the spec)